tests/unit/Design.py

```diff
@@ -140,7 +140,8 @@ def test_Files(self):
 
 
 class Validate(TestCase):
-	def test_Design(self):
+	@staticmethod
+	def test_Design():
```

Comment:
I think it's not good to convert tests to `staticmethod`s.

Have you checked of pytest is finding these tests?